data/mods/gen9ssb/scripts.ts

```diff
@@ -1085,6 +1085,7 @@ export const Scripts: ModdedBattleScriptsData = {
 					const isCharging = move.flags['charge'] && !this.volatiles['twoturnmove'] &&
 						!(move.id.startsWith('solarb') && ['sunnyday', 'desolateland'].includes(this.effectiveWeather())) &&
 						!(move.id === 'fruitfullongbow' && ['sunnyday', 'desolateland'].includes(this.effectiveWeather())) &&
+						!(move.id === 'praisethemoon' && ['gravity'].includes(this.effectiveWeather())) &&
```

Comment:
```suggestion
						!(move.id === 'praisethemoon' && this.battle.field.getPseudoWeather('gravity')) &&
```
Gravity is a Pseudo weather, not an actual weather condition.